### ISSUE TITLE
Remote Assistance freezes after a short time

### ISSUE THREAD

**Issue by myheroyuki** [FreeRDP/FreeRDP#8610]
**Describe the bug**
When looking into remote assistance capabilities for Remmina, I noticed that soon after connecting the screen would freeze and stop updating any changes from the remote Windows box. It works fine initially, but after a few seconds to a minute everything stops, even though it still looks as though it is connected. I didn't see any other open issues with a similar problem.

**To Reproduce**
I built FreeRDP myself using the same steps I use to build it for testing with Remmina:
1. git clone --branch stable-2.0 https://github.com/FreeRDP/FreeRDP.git
2. cd FreeRDP
3. mkdir build; cd build
4. `cmake .. -DCMAKE_BUILD_TYPE=Debug -DWITH_SSE2=ON -DWITH_CUPS=on -DWITH_ICU=on -DWITH_PULSE=on -DCMAKE_INSTALL_PREFIX:PATH=/opt/remmina_devel/freerdp .`
5. make && sudo make install

From there, I try to attempt to a windows box using the following command:

`WLOG_LEVEL=DEBUG xfreerdp /home/user/Documents/Invitation.msrcIncident /v:<windows_ip> /assistance:<Password>

**Expected behavior**
The connection should work until it is manually stopped instead of freezing itself 

**Screenshots**

**Application details**
* FreeRDP version: `This is FreeRDP version 2.9.0 (2.9.0)`
* Command line used: Default Ubuntu
* Output of `xfreerdp /buildconfig`:
`This is FreeRDP version 2.9.0 (2.9.0)
Build configuration: BUILD_TESTING=OFF BUILTIN_CHANNELS=ON HAVE_AIO_H=1 HAVE_EXECINFO_BACKTRACE=1 HAVE_EXECINFO_BACKTRACE_SYMBOLS=1 HAVE_EXECINFO_BACKTRACE_SYMBOLS_FD=1 HAVE_EXECINFO_H=ON HAVE_EXECINFO_HEADER=1 HAVE_FCNTL_H=1 HAVE_GETLOGIN_R=1 HAVE_GETPWUID_R=1 HAVE_INTTYPES_H=1 HAVE_JOURNALD_H=TRUE HAVE_MATH_C99_LONG_DOUBLE=1 HAVE_PIXMAN_REGION=OFF HAVE_POLL_H=1 HAVE_PTHREAD_MUTEX_TIMEDLOCK=ON HAVE_PTHREAD_MUTEX_TIMEDLOCK_LIB=1 HAVE_PTHREAD_MUTEX_TIMEDLOCK_SYMBOL= HAVE_SYSLOG_H=1 HAVE_SYS_EVENTFD_H=1 HAVE_SYS_FILIO_H= HAVE_SYS_MODEM_H= HAVE_SYS_SELECT_H=1 HAVE_SYS_SOCKIO_H= HAVE_SYS_STRTIO_H= HAVE_SYS_TIMERFD_H=1 HAVE_TM_GMTOFF=1 HAVE_UNISTD_H=1 HAVE_XI_TOUCH_CLASS=1 WITH_ALSA=ON WITH_CAIRO=OFF WITH_CCACHE=ON WITH_CHANNELS=ON WITH_CLANG_FORMAT=ON WITH_CLIENT=ON WITH_CLIENT_AVAILABLE=1 WITH_CLIENT_CHANNELS=ON WITH_CLIENT_CHANNELS_AVAILABLE=1 WITH_CLIENT_COMMON=ON WITH_CLIENT_INTERFACE=OFF WITH_CUPS=on WITH_DEBUG_ALL=OFF WITH_DEBUG_CAPABILITIES=OFF WITH_DEBUG_CERTIFICATE=OFF WITH_DEBUG_CHANNELS=OFF WITH_DEBUG_CLIPRDR=OFF WITH_DEBUG_DVC=OFF WITH_DEBUG_KBD=OFF WITH_DEBUG_LICENSE=OFF WITH_DEBUG_MUTEX=OFF WITH_DEBUG_NEGO=OFF WITH_DEBUG_NLA=OFF WITH_DEBUG_NTLM=OFF WITH_DEBUG_RAIL=OFF WITH_DEBUG_RDP=OFF WITH_DEBUG_RDPDR=OFF WITH_DEBUG_RDPEI=OFF WITH_DEBUG_RDPGFX=OFF WITH_DEBUG_REDIR=OFF WITH_DEBUG_RFX=OFF WITH_DEBUG_RINGBUFFER=OFF WITH_DEBUG_SCARD=OFF WITH_DEBUG_SND=OFF WITH_DEBUG_SVC=OFF WITH_DEBUG_SYMBOLS=OFF WITH_DEBUG_THREADS=OFF WITH_DEBUG_TIMEZONE=OFF WITH_DEBUG_TRANSPORT=OFF WITH_DEBUG_TSG=OFF WITH_DEBUG_TSMF=OFF WITH_DEBUG_TSMF=OFF WITH_DEBUG_TSMF_AVAILABLE=0 WITH_DEBUG_URBDRC=OFF WITH_DEBUG_WND=OFF WITH_DEBUG_X11=OFF WITH_DEBUG_X11_CLIPRDR=OFF WITH_DEBUG_X11_LOCAL_MOVESIZE=OFF WITH_DEBUG_XV=OFF WITH_DSP_EXPERIMENTAL=OFF WITH_DSP_FFMPEG=OFF WITH_EVENTFD_READ_WRITE=1 WITH_FAAC=OFF WITH_FAAD2=OFF WITH_FFMPEG=TRUE WITH_FFMPEG=TRUE WITH_GFX_H264=ON WITH_GPROF=OFF WITH_GSM=OFF WITH_GSSAPI=OFF WITH_ICU=on WITH_INTERNAL_MD4=OFF WITH_INTERNAL_MD5=OFF WITH_IPP=OFF WITH_JPEG=OFF WITH_LAME=OFF WITH_LIBRARY_VERSIONING=ON WITH_LIBSYSTEMD=ON WITH_MACAUDIO=OFF WITH_MACAUDIO=OFF WITH_MACAUDIO_AVAILABLE=0 WITH_MANPAGES=ON WITH_MBEDTLS=OFF WITH_OPENCL=OFF WITH_OPENH264=OFF WITH_OPENSLES=OFF WITH_OPENSSL=ON WITH_OSS=ON WITH_PCSC=OFF WITH_PROFILER=OFF WITH_PULSE=on WITH_SAMPLE=OFF WITH_SANITIZE_ADDRESS=OFF WITH_SANITIZE_ADDRESS_AVAILABLE=1 WITH_SANITIZE_MEMORY=OFF WITH_SANITIZE_MEMORY_AVAILABLE=1 WITH_SANITIZE_THREAD=OFF WITH_SANITIZE_THREAD_AVAILABLE=1 WITH_SERVER=OFF WITH_SERVER_INTERFACE=ON WITH_SMARTCARD_INSPECT=OFF WITH_SOXR=OFF WITH_SSE2=ON WITH_SWSCALE=OFF WITH_THIRD_PARTY=OFF WITH_VAAPI=OFF WITH_VALGRIND_MEMCHECK=OFF WITH_VALGRIND_MEMCHECK_AVAILABLE=1 WITH_VERBOSE_WINPR_ASSERT=ON WITH_WAYLAND=ON WITH_WINPR_TOOLS=ON WITH_X11=ON WITH_XCURSOR=ON WITH_XEXT=ON WITH_XFIXES=ON WITH_XI=ON WITH_XINERAMA=ON WITH_XKBFILE=ON WITH_XRANDR=ON WITH_XRENDER=ON WITH_XSHM=ON WITH_XV=ON WITH_ZLIB=ON
Build type:          Debug
CFLAGS:               -fPIC -Wall -Wno-unused-result -Wno-unused-but-set-variable -Wno-deprecated-declarations -fvisibility=hidden -Wimplicit-function-declaration -Wredundant-decls -g -fno-omit-frame-pointer -DWINPR_DLL
Compiler:            GNU, 9.4.0
Target architecture: x64
`
* OS version connecting to (server side): Windows 10
* If available the log output from a run with `/log-level:trace 2>&1 | tee log.txt` : 
[log.txt](https://github.com/FreeRDP/FreeRDP/files/10411694/log.txt)

**Environment (please complete the following information):**

 Description:	Ubuntu 20.04.4 LTS x86_64

**Additional context**
I tested on an Ubuntu 22 as well, and tried connecting to a second windows 10 box too, and had the same result, so I don't think the issue was with my set-up, but it would be nice if the issue was as simple as me building the tool incorrectly.

**Comment by SteveParson** [comment]
@myheroyuki How long does it typically take until it freezes? I'm trying to reproduce your issue on FreeRDP 2.10.0

**Comment by akallabeth** [comment]
@myheroyuki could you test with `flatpak com.freerdp.FreeRDP`? that is current stable compiled by us (FreeRDP team).
I fail to reproduce any issue here against a windows 10 test machine (did let it run for more than an hour, observation only, no remote control) but I´m on 10 GBit lan here, so that might influence.

**Comment by myheroyuki** [comment]
Unfortunately, I haven't been able to connect at all using FreeRDP on the command line, using remote assistance or just a regular RDP connection, either with the newest flatpak or the standard install I already had.  Trying to connect with `/u:<user> /p:<password> /v:<server>` fails with the following error

`[20:44:10:263] [26639:26640] [WARN][com.freerdp.core.nla] - SPNEGO received NTSTATUS: STATUS_LOGON_FAILURE [0xC000006D] from server
[20:44:10:263] [26639:26640] [ERROR][com.freerdp.core] - nla_recv_pdu:freerdp_set_last_error_ex ERRCONNECT_LOGON_FAILURE [0x00020014]
[20:44:10:263] [26639:26640] [ERROR][com.freerdp.core.rdp] - rdp_recv_callback: CONNECTION_STATE_NLA - nla_recv_pdu() fail
[20:44:10:263] [26639:26640] [ERROR][com.freerdp.core.transport] - transport_check_fds: transport->ReceiveCallback() - -1
[20:44:10:263] [26639:26640] [DEBUG][com.freerdp.core.rdp] - transport_check_fds() - -1
[20:44:10:263] [26639:26640] [DEBUG][com.freerdp.core.connection] - rdp_client_transition_to_state CONNECTION_STATE_NLA --> CONNECTION_STATE_INITIAL
`


I'm not sure what changed as this was working for me before, so I have some more digging to do, but just haven't had much time to look into it yet. Interestingly, I am still able to have totally successful RDP sessions when I use Remmina

**Comment by akallabeth** [comment]
@myheroyuki well, your credentials do not work for the target... `STATUS_LOGON_FAILURE` is a error code from `NLA` authentication

**Comment by myheroyuki** [comment]
That was my initial reaction as well, but I triple checked that were correct. They were identical to the credentials being used in Remmina that were working fine. 

When I re-ran the command and added the `/sec:tls` flag, a window opened showing the windows login screen that was displaying an incorrect login credentials message and asking me to try again. Re-typing the exact same credentials I used when running the command worked and I was able log in fine
